### DISCUSSION TITLE
Is fuse_loop_mt() blocking?

### DISCUSSION THREAD

**Discussion by infeo** [libfuse/libfuse#826] [Q&A] [answered]
This might have an obvious answer, but since in the docs no entry for `fuse_loop_mt` exist and it's single threaded sibling also does not mention to be blocking directly and the for lowlevel functions `fuse_session_loop` and `fuse_session_loop_mt` only [the first is documented as blocking](https://github.com/libfuse/libfuse/blob/1f0dfae4084577997291bb0e1b94aeff89a5e70f/include/fuse_lowlevel.h#L2063C5-L2063C5), i just want to have some clarity.

**Comment by trapexit** [comment, marked as answer]
https://github.com/libfuse/libfuse/blob/master/lib/fuse_loop_mt.c#L365

Yes, it blocks.
